collector/collector_test.go

```diff
@@ -763,7 +758,7 @@ var (
 			{"source": "snmp", "name": "snmp part", "description": "part received through SNMP request", "OID": ".1.3.6.1.2.1.1.9.1.3"},
 			{"source": "string", "string": "value"}
 		  ],
-		  "OID": ".1.3.6.1.2.1.1.9.1.4",
```

Comment:
@michep I've seen your comment but are you able to identify change in the code which triggers this change in tests?